platform/ios/ios.xcodeproj/xcshareddata/xcschemes/dynamic.xcscheme

```diff
@@ -87,7 +87,7 @@
       </MacroExpansion>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
+      buildConfiguration = "RelWithDebInfo"
```

Comment:
Why is this changing?

Reply:
This is a hold-over from previous testing - however, it's easier to get meaningful results when profiling with symbols.